doc/user/data/sql_funcs.yml

```diff
@@ -833,41 +868,41 @@
+    - signature: "has_connection_privilege([role: text or oid,] connection: text or oid, privilege: text) -> bool"
       description: >-
         Reports whether the role with the specified role name or OID has the privilege on
         the connection with the specified connection name or OID. If the role is omitted then
         the `current_role` is assumed.
-    - signature: 'has_database_privilege([role: text or oid,] database: text or oid, privilege: text) -> bool'
+    - signature: "has_database_privilege([role: text or oid,] database: text or oid, privilege: text) -> bool"
       description: >-
         Reports whether the role with the specified role name or OID has the privilege on
         the database with the specified database name or OID. If the role is omitted then
         the `current_role` is assumed.
-    - signature: 'has_schema_privilege([role: text or oid,] schema: text or oid, privilege: text) -> bool'
+    - signature: "has_schema_privilege([role: text or oid,] schema: text or oid, privilege: text) -> bool"
       description: >-
         Reports whether the role with the specified role name or OID has the privilege on
         the schema with the specified schema name or OID. If the role is omitted then
         the `current_role` is assumed.
-    - signature: 'has_secret_privilege([role: text or oid,] secret: text or oid, privilege: text) -> bool'
+    - signature: "has_secret_privilege([role: text or oid,] secret: text or oid, privilege: text) -> bool"
       description: >-
         Reports whether the role with the specified role name or OID has the privilege on
         the secret with the specified secret name or OID. If the role is omitted then
         the `current_role` is assumed.
-    - signature: 'has_system_privilege([role: text or oid,] privilege: text) -> bool'
+    - signature: "has_system_privilege([role: text or oid,] privilege: text) -> bool"
       description: >-
         Reports whether the role with the specified role name or OID has the system privilege.
         If the role is omitted then the `current_role` is assumed.
-    - signature: 'has_table_privilege([role: text or oid,] relation: text or oid, privilege: text) -> bool'
+    - signature: "has_table_privilege([role: text or oid,] relation: text or oid, privilege: text) -> bool"
       description: >-
         Reports whether the role with the specified role name or OID has the privilege on
         the relation with the specified relation name or OID. If the role is omitted then
         the `current_role` is assumed.
-    - signature: 'has_type_privilege([role: text or oid,] type: text or oid, privilege: text) -> bool'
+    - signature: "has_type_privilege([role: text or oid,] type: text or oid, privilege: text) -> bool"
       description: >-
         Reports whether the role with the specified role name or OID has the privilege on
         the type with the specified type name or OID. If the role is omitted then
```

Comment:
Could you back out whatever autoformatting happened on this file without losing your changes? It's approximately impossible to see what you added at the moment!

Reply:
Whoops. Yeah, I'll revert that.